src/core/client/ui/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,70 @@
+import cn from "classnames";
+import React, { ReactNode } from "react";
+import { withStyles } from "talk-ui/hocs";
+import * as styles from "./Tab.css";
+
+export interface TabProps {
+  /**
+   * Convenient prop to override the root styling.
+   */
+  className?: string;
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes: typeof styles;
+  /**
+   * The id/name of the tab
+   */
+  tabId: string;
+  /**
+   * Active status
+   */
+  active?: boolean;
+  /**
+   * Color style variant
+   */
+  color?: "primary" | "secondary";
```

Comment:
I'm in favor of using `variant`, since these are not colors. But we need to be consistent, the `Button` component uses this as `color`property. 

Will change it in favour of `variant`.